packages/integration_test/ios/Classes/IntegrationTestIosTest.h

```diff
@@ -4,23 +4,40 @@
 
 #import <Foundation/Foundation.h>
 
+NS_ASSUME_NONNULL_BEGIN
+
+@protocol FLTIntegrationTestScreenshotDelegate;
+
 @interface IntegrationTestIosTest : NSObject
 
-- (BOOL)testIntegrationTest:(NSString **)testResult;
+- (instancetype)initWithScreenshotDelegate:(nullable id<FLTIntegrationTestScreenshotDelegate>)delegate NS_DESIGNATED_INITIALIZER;
+
+- (BOOL)testIntegrationTest:(NSString *_Nullable *_Nullable)testResult;
```

Comment:
Could we get a comment on this weird API describing its behavior while you are touching it? (Or even better, could we make this an idiomatic ObjC method that returns either an NSString or nil?)

Reply:
This API is very weird (I just added the nullability), and the `INTEGRATION_TEST_IOS_RUNNER` pragma is downright Swift-hostile...